src/video/vga_state.c

```diff
@@ -75,6 +75,11 @@ void vga_state_mark_remaps_flushed(void) {
     state.dirty_remaps = false;
 }
 
+void vga_state_mark_invalidate_render_plugin(void) {
```

Comment:
vga_state_mark_dirty() instead ? Seems unnecessary to make this specific for renderer plugin validation.